pysearch/templates/results.jinja2

```diff
@@ -0,0 +1,135 @@
+{% extends "layout.jinja2" %}
+
+{% block title %}Results{% endblock %}
+
+{% block nav %}
+
+<nav class="navbar ">
+  <div class="container-fluid">
+    <div class="collapse navbar-collapse" id="myNavbar">
+
+    <div class="col-lg-9"></div>
+      <ul class="nav navbar-nav topBar">
+        <li class="active"><a href="/">Home</a></li>
+        <li><a href="/about">About</a></li>
+      </ul>
+    <div class="col-lg-2"></div>
+
+    </div>
+  </div>
+</nav>
+{%endblock%}
+
+
+{% block body %}
+
+<div class="jumbotron ">
+  <div class="container text-center">
+    <h1>Pysearch Results</h1>      
+  </div>
+</div>
+{% if RESULTS %}
+    {% set count = 1 %}
+    {% set top_score = RESULTS[0].score%}
+    {% for result in RESULTS %}
+    {% set perc = result.score/top_score * 100 %}
+  
+  {% if loop.index == 1 %}
+    <div class="row">
+    <div class="col-lg-3"></div>
+    <div class="col-lg-6">
+      <div class="panel panel-primary">
+
+        <div class="panel-heading search-url">SEARCH URL:</div>
+        <div class="panel-body search-url"><a href={{ web_page }}>{{ web_page }}</a></div>
+        <div class="panel-footer search-url">Matched {{RESULTS|length - 1}} Sites</div>
+        
+      </div>
+      <div class="col-lg-3"></div>
+    </div>
+  </div>
+  {%else%}
+  
+  
+  <div class="row">
+  <div class="col-lg-2"></div>
+    <div class="col-lg-8">
+      <div class="panel panel-primary">
+
+        <div class="panel-heading">{{loop.index - 1}}.  <a class= "result-heading"href={{result.url}}>{{result.url}}</a></div>
+        <div class="panel-body">Matched with a score of: {{result.score}}</div>
+        <div class="panel-footer">{{perc|round(2)}} Percent Match</div>
+        
+      </div>
+    </div>
+  </div>
+  {% set count = count + 1 %}
+  {%endif%}
+  {%endfor%}
+  {% else %}
+   <div class="row">
+    <div class="col-lg-3"></div>
+    <div class="col-lg-6">
+      <div class="panel panel-primary">
+
+        <div class="panel-heading search-url"></div>
+        <div class="panel-body search-url">NO RESULTS FOUND</div>
+        <div class="panel-footer search-url"></div>
+        
+      </div>
+      <div class="col-lg-3"></div>
+    </div>
+  </div>
+ {%endif%} 
+</div><br>
+
+
+
+{# 
+<table id="result-list">
+
+    <tr>
+    <h1>Results:</h1>
+        <th>Title</th>
+        <th>Url</th>
+        <th>Content</th>
+    </tr>
+
+    {% for result in RESULTS %}
+    <tr>
+        <th></th>
+        <th>{{ result['page_url'] }}</th> 
+        <th>"This apple pie is amazing! ..."</th>
+        <td>Result: <a href="{{ result }}">{{ result }}</a></td>
+    </tr>
+    {% endfor %}
+
+</table>
+ #}
```

Comment:
This is meant to be commented code. It should be removed if it's not being used.